bl/evaluation/evaluator.go

```diff
@@ -43,15 +41,16 @@ type EvaluationResults struct {
 }
 
 func (e *Evaluator) CreateEvaluation(cliId string, cliVersion string, k8sVersion string, policyName string, ciContext *ciContext.CIContext) (*cliClient.CreateEvaluationResponse, error) {
+	osInfo := NewOSInfo()
```

Comment:
Shouldn't we use a stub for this, instead of using the actual value?
Something like: 

```suggestion
	osInfo := &OSInfo{ OS: "darwin", PlatformVersion: "1.2.3", KernelVersion:   "4.5.6", }
```